saleor/graphql/order/tests/test_order.py

```diff
@@ -1302,6 +1318,73 @@ def test_order_query_shows_non_draft_orders(
     assert len(edges) == Order.objects.non_draft().count()
 
 
+def test_order_granted_refunds_query(
```

Comment:
IMO we should have a similar test for querying `grantedRefunds` with `app` assigned instead of a user.

Reply:
Thanks, done.